internal/cli/usercmd/app.go

```diff
@@ -526,3 +526,45 @@ func (c *EpinioClient) printReplicaDetails(app models.App) error {
 
 	return nil
 }
+
+// AppRestage restage an application
+func (c *EpinioClient) AppRestage(appName string) error {
+	log := c.Log.WithName("AppRestage").WithValues("Namespace", c.Settings.Namespace, "Application", appName)
+	log.Info("start")
+	defer log.Info("return")
+
+	c.ui.Note().
+		WithStringValue("Namespace", c.Settings.Namespace).
+		WithStringValue("Application", appName).
+		Msg("Restaging application")
+
+	if err := c.TargetOk(); err != nil {
+		return err
+	}
+
+	log.V(1).Info("restaging application")
+
+	app, err := c.API.AppShow(c.Settings.Namespace, appName)
+	if err != nil {
+		return err
+	}
+
+	if app.Origin.Kind == models.OriginContainer {
+		return nil
```

Comment:
IMHO this should print a note, i.e. `c.ui.Note().Msg("Unable to restage container-based application")`, or similar.